src/validators.ts

```diff
@@ -165,3 +165,10 @@ export function map<K, V>(
     return result;
   };
 }
+
+export function multiType<F, S>(
+  first: Validator<F>,
+  second: Validator<S>,
+): Validator<F | S> {
+  return (val: any): F | S => first(val) && second(val);
```

Comment:
You are right, I added `try` and `catch` construct, @Mossop  PTAL. Also, because I removed `UpdateBugContentSpec` and `CreateBugContentSpec`, there is no need for such a validator for now. So, we can leave it and maybe it can become useful later or we can probably drop it.